docs/installing.rst

```diff
@@ -55,8 +74,69 @@ Install behind a proxy::
   python get-pip.py --proxy="[user:passwd@]proxy.server:port"
 
 
-Upgrade pip
------------
+Installing with Linux Package Managers
+--------------------------------------
+
+Fedora
+~~~~~~
+
+To install the Fedora supported version of pip:
+
+* < Fedora 23:
+
+ * Python 2: ``sudo yum install python-pip``
+ * Python 3: ``sudo yum install python3-pip``
+
+* >= Fedora 23:
+
+ * Python 2: ``sudo dnf install python-pip``
+ * Python 3: ``sudo dnf install python3-pip``
+
+To get newer versions of pip (and also setuptools and wheel), you can enable the
+"unofficial" `PyPA Copr Repo <https://copr.fedoraproject.org/coprs/pypa/pypa/>`_
+using `these instructions
+<https://fedorahosted.org/copr/wiki/HowToEnableRepo>`__, and run the same
+commands as above.
+
+
+CentOS/RHEL
+~~~~~~~~~~~
+
+CentOS and RHEL don't offer pip in their core repositories.
```

Comment:
@ncoghlan is there a place to browse the RHEL Collections?  i.e. something different than CentOS's softwarecollections.org?